main.py

```diff
@@ -1,10 +1,44 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+    for player_ in players:
+
+        player = players[player_]
+
+        if player.get("guild") is not None:
+            guild, created = Guild.objects.get_or_create(
```

Comment:
You could omit creating `created` variable with `_` since it is not used in the logic.